Sources/ShortcutRecorder/SRShortcut.m

```diff
@@ -421,7 +421,7 @@ - (void)encodeWithCoder:(NSCoder *)aCoder
 + (instancetype)new
 {
     [self doesNotRecognizeSelector:_cmd];
-    return nil;
+    return [super new];
```

Comment:
This change is undesirable since we don't want to instantiate any object here. An exception that leads to crash is desirable here since this is a programmer's error.

Reply:
Well it still does throw an exception, basically this line is dead code. But newer Xcode versions warn about returning `nil` from `new`, I thought it's better to fix this than to disable the warning in project settings, no?